scripts/bootstrap.py

```diff
@@ -69,12 +69,12 @@
 except ImportError:
     ez = {}
     if USE_DISTRIBUTE:
-        exec urllib2.urlopen('http://python-distribute.org/distribute_setup.py'
-                         ).read() in ez
+        exec(urllib2.urlopen('http://python-distribute.org/distribute_setup.py'
```

Comment:
These fixes were added automatically by the `futurize` command. The script still isn't quite python 3 compatible, but seems to be broken anyway, since the link 404s.